code/modules/mob/living/carbon/slime/slime.dm

```diff
@@ -938,6 +972,33 @@ mob/living/carbon/slime/var/temperature_resistance = T0C+75
 		M.cores = 3
 		qdel (src)
 
+
+/obj/item/weapon/slimenutrient
+	name = "Slime Nutrient"
+	desc = "A potent chemical mix that is a great nutrient for slimes."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bottle12"
+	var/Uses = 2
+
+/obj/item/weapon/slimenutrient/attack(mob/living/carbon/slime/M as mob, mob/user as mob)
+	if(!istype(M, /mob/living/carbon/slime))//If target is not a slime.
```

Comment:
You can leave out the second argument of the `istype` because BYOND knows what type of variable `M` is.